### PR TITLE
Use JENKINS_SECRET and JENKINS_NAME env vars if present

### LINKED ISSUE #3: Feature Request: Env variables for secret and slave name
Like `JENKINS_URL` it should be possible to specify `SLAVE_SECRET` and `SLAVE_NAME`. This should make it easier to provision slave templates and avoid mistakes with entrypoint and command.

### PR DESCRIPTION
**Pull request by carlossg**
Fix #3

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@reviewbybees

File: jenkins-slave [hunk 1]
```diff
@@ -22,10 +22,12 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-# Usage jenkins-slave.sh [options] -url http://jenkins SECRET SLAVE_NAME
+# Usage jenkins-slave.sh [options] -url http://jenkins [SECRET SLAVE_NAME]
 # Optional environment variables :
 # * JENKINS_TUNNEL : HOST:PORT for a tunnel to route TCP traffic to jenkins host, when jenkins can't be directly accessed over network
 # * JENKINS_URL : alternate jenkins URL
+# * JENKINS_SECRET : agent secret, if not set as an argument
+# * JENKINS_NAME : agent name, if not set as an argument
```
Comment (jglick) [inline jenkins-slave]: `JENKINS_SLAVE_NAME` would be clearer. Or better yet, `JENKINS_AGENT_NAME` with `SLAVE_NAME` replaced by `AGENT_NAME` above.

Comment (carlossg) [inline jenkins-slave]: @oleg-nenashev you approved, so WDYT ?

Comment (carlossg) [inline jenkins-slave]: IIRC `name` and `secret` are the var names used in remoting

Comment (oleg-nenashev) [inline jenkins-slave]: Yes, +1 for `JENKINS_AGENT_NAME`

**Comment by oleg-nenashev** [comment]
@reviewbybees done

**Pull request merged**